src/devices/fineoffset.c

```diff
@@ -380,12 +387,14 @@ static int fineoffset_WH0290_callback(r_device *decoder, bitbuffer_t *bitbuffer)
     uint8_t id        = b[1];
     int pm25          = (b[2] & 0x3f) << 8 | b[3];
     int pm100         = (b[4] & 0x3f) << 8 | b[5];
-
+    int battery_bars  = (b[2] & 0x40) >> 4 | (b[4] & 0xC0) >> 6; //out of 5
+    float battery_ok  = battery_bars / 5.0; //convert out of 5 bars to 0 (0 bars) to 1 (5 bars)
```

Comment:
`* 0.2f` is preferred

Reply:
Fix committed.

Reply:
sorry, I meant `float battery_ok  = battery_bars * 0.2f` the format as `%.1f` is ok ;)

Reply:
my mistake I'll fix

Reply:
Commited Fix - reverted the format change, back to `%.1f` and changed the `/ 5.0` to `* 0.2f`
Rebuilt and retested successfully after these changes.